presto-product-tests/src/main/resources/sql-tests/testcases/table_with_header.sql

```diff
@@ -2,7 +2,7 @@
 --! name: simple_scan
 SELECT count(*) FROM table_with_header
 --!
-32768
+34816
```

Comment:
One more thought: if you used `sum(id)` instead of `count(*)`, you would know that some data is actually read.